unit/test/Get-VSTeamPullRequest.Tests.ps1

```diff
@@ -203,6 +203,13 @@ Describe 'VSTeamPullRequest' {
 
          $pr.reviewStatus | Should -Be "Rejected"
       }
+
+      It 'with RepositoryId and IncludeCommits' {
+         Get-VSTeamPullRequest -Id 101 -RepositoryId "93BBA613-2729-4158-9217-751E952AB4AF" -IncludeCommits
+         Should -Invoke Invoke-RestMethod -Exactly -Scope It -Times 1 -ParameterFilter {
+            $Uri -eq "https://dev.azure.com/test/_apis/git/repositories/93bba613-2729-4158-9217-751e952ab4af/pullRequests/101?api-version=$(_getApiVersion Git)&includeCommits=True"
```

Comment:
Could you please split the assert for the URL like here: 
https://github.com/DarqueWarrior/vsteam/blob/c6ea62388e8c66e843ce73036ddfc4db82292071/unit/test/Get-VSTeamWorkItem.Tests.ps1#L51

Reply:
Done - should be checking all the relevant parts now separately.